lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -67,6 +67,7 @@ def self.canonicalize_url(driver_url, config)
     # Load balancer methods
     def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
       lb_options = AWSResource.lookup_options(lb_options || {}, managed_entry_store: lb_spec.managed_entry_store, driver: self)
+      lb_attributes = lb_options.delete(:load_balancer_attributes)
```

Comment:
Can you add a comment saying `We delete the options here because they are not valid in the create call, must be applied afterwords` - I could see myself asking why we are deleting this in about 2 months